doc/source/conf.py

```diff
@@ -10,12 +10,15 @@
 
 import sleepecg
 
-# -- Project information --------------------------------------------------
+# -- Path setup ----------------------------------------------------------------------------
+sys.path.insert(0, os.path.abspath("../../sleepecg"))
```

Comment:
Why is this necessary?

Reply:
Seems like it isn't. I read that [here](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/sphinx-config.html#autodoc-configuration), so I went ahead and added it. Do you know why it is not required?

Reply:
Before building the docs locally or on RTD, the package is installed, so it's available in the pythonpath anyway.